src/fides/api/alembic/migrations/versions/14acee6f5459_translation_data.py

```diff
@@ -430,11 +449,11 @@ def determine_needed_experience_configs(bind):
         component_mappings = NOTICES_TO_EXPERIENCE_CONFIG.get(notice_key, [])
         if component_mappings:
             for component_mapping in component_mappings:
-                if (
-                    component_mapping.get(ComponentType.Overlay)
-                    == DefaultExperienceConfigTypes.CANADA_BANNER_MODAL
-                ):
-                    # If this is a Canadian Banner and Modal, skip updating regions, and just use OOB
+                if component_mapping.get(ComponentType.Overlay) in [
+                    DefaultExperienceConfigTypes.CANADA_BANNER_MODAL,
+                    DefaultExperienceConfigTypes.QUEBEC_BANNER_MODAL,
+                ]:
```

Comment:
you don't actually need to change this, this type of thing is just a pet peeve of mine so i can't help but write it :trollface: 

```suggestion
                if component_mapping.get(ComponentType.Overlay) in {
                    DefaultExperienceConfigTypes.CANADA_BANNER_MODAL,
                    DefaultExperienceConfigTypes.QUEBEC_BANNER_MODAL,
                }:
```

Reply:
why is it a pet peeve? explain! 😄

Reply:
well isn't `in` a constant-time operation on a set, and a linear-time operation on a list? so a set feels like the _right_ data structure here! but now i'm starting to doubt myself? 🤔 

of course there's no practical difference in this case. it's just that i get paranoid about using the wrong data structure and implying that there's a _reason_ a list is here, i.e. that the order of these elements matters. and python is nice in allowing either to be declared inline so cleanly - so it just feels "right" to use a set here!

this is so pedantic, i'm reluctant to waste brain space talking about it... but you asked! 😄

Reply:
sorry I commented before refreshing and hadn't seen your comment!

Reply:
no that makes sense thank you for explaining, I'll adapt this too

Reply:
> faster lookups even for sets of two?

it's a totally fair response :)

Reply:
I hadn't even considered at all the point about a list implying ordering mattering, great point

Reply:
i mean, it's really quite a stretch, especially in this case. i guess my pet peeve is that generally i tend to be very deliberate about the data structures i use (whether i generally get them _right_ though is another story 😄) - so something like this catches my eye. but i think it's often more trouble than it's worth!

Reply:
yep good reminder 💪